documentation/toasts.html

```diff
@@ -0,0 +1,147 @@
+                </li>
+            </ul>
+        </div>
+        <div class="main-content p-5">
+            <div class="component-details">
+                <h3 class="text-bold mb-5">TOASTS</h3>
+                <p class="mb-5">
+                    Toast help display time-sensitive, real-time updates to users
+                </p>
+                <h5 class="mb-4">COMPONENT DEMO (click on the button to see the results)</h5>
+                <div class="component-demonstration p-6">
+                    <button class="btn btn-secondary snackbar-button">BASELINE</button>
+                    <button class="btn btn-secondary snackbar-button">LEADING</button>
+                </div>
+                <div class="gentle-snackbar-popup baseline">
+                    <p class="gentle-snackbar-text">Can't send photos.Retry in 5 seconds.</p>
+                    <div class="gentle-snackbar-actions">
+                        <a class="gentle-snackbar-link" href="">UNDO</a>
+                        <i class="fas fa-times close-icon"></i>
+                    </div>
+                </div>
+                <div class="gentle-snackbar-popup leading">
+                    <p class="gentle-snackbar-text">Your photo has been archived.</p>
+                    <div class="gentle-snackbar-actions">
+                        <a class="gentle-snackbar-link" href="">UNDO</a>
+                        <i class="fas fa-times close-icon"></i>
+                    </div>
+                </div>
+                <div class="code-demonstration">
+                    <script
+                        src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fdev-enforced%2FGentle_UI%2Fblob%2Fe061c3e3bc6ce355711cf5a3522fc37155e21b5f%2Fdocumentation%2Ftoasts.html%23L98-L111&style=atom-one-dark&showBorder=on&showCopy=on"></script>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+        const btns = document.querySelectorAll(".snackbar-button");
+        const baselineSnackbar = document.querySelector(".baseline");
+        const leadingSnackbar = document.querySelector(".leading");
+        const stackedSnackbar = document.querySelector(".stacked");
+
+        function showSnackbar(snackbarType) {
+            if (snackbarType === "BASELINE") {
+                baselineSnackbar.style.display = "flex";
+                setTimeout(() => {
+                    baselineSnackbar.style.display = "none";
+                }, 2000);
+            } else if (snackbarType === "LEADING") {
+                leadingSnackbar.style.display = "flex";
+                setTimeout(() => {
+                    leadingSnackbar.style.display = "none";
+                }, 2000);
+            } else if (snackbarType === "STACKED") {
+                stackedSnackbar.style.display = "flex";
+                setTimeout(() => {
+                    stackedSnackbar.style.display = "none";
+                }, 2000);
```

Comment:
One Suggestion, good to have scripts in the separate file

Reply:
Yes @fahd23 all scripts have been moved to a common folder

Reply:
Here [36f6c49](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/36f6c49a14b73c836aea7987077afe09aa911c6b)